prime-router/src/test/kotlin/history/azure/DeliveryFacadeTests.kt

```diff
@@ -262,6 +262,54 @@ class DeliveryFacadeTests {
         assertThat(facilities.first().location).isEqualTo(result.first().location)
     }
 
+    @Test
+    fun `test findDeliveryFacilitiesBulk`() {
```

Comment:
Please resolve this or otherwise comment why it was not resolved.

Reply:
This one we can't really test much on since the facade method is just relaying data back and forth. The surrounding tests should be doing the coverage.